run-integration-test/action.yml

```diff
@@ -103,8 +103,11 @@ runs:
           fi
         fi
 
+        TEST_PARALLELISM=$(yq -er '.parallelism // 2' ./tests/infrastructure.yaml)
+
```

Comment:
note: Can we move this up to where we read in all the other options from the config file?

Reply:
Done 17f66b09ca4dffa0db4274e3a65f5b9d5e25a60f